Add blogpost about 2023 edition (#125)
jsheroes/website: src/content/blog/my-experience-at-jsheroes-2023.md

---
author: alex-pavaloi
author_category: guest-writers
title: My Experience at JSHeroes 2023
published: Dec 8, 2023
tags: ["general"]
summary: Hi! I'm Alex and this blog post is about my experience at JS Heroes 2023.
---

After a couple of years without in-person conferences, I was really glad to return to JSHeroes for a 2-day event. So I'm gonna use this blog post to share my experience with you.

> I have to start with the MC - Anjana Vakil - which I think was a perfect fit for the role. She was so full of energy and enthusiasm that she really got us pumped for the conference. I wish I could embed some video snippets here so you could see her contagious energy for yourself.

Reviewer: You can link the after movie maybe? (https://www.youtube.com/watch?v=N6qr6L37sl8) she's quite visible at the beginning :) youtube link can also be embedded I think

Author: good idea. Linked the movie, found a part at the beginning that really fits what the paragraph is about.

Moving on to the technical side, the conference had a big diversity in the topics covered. Things like CSS, performance, accessibility, realtime apps, state management and AI-powered experiences were just a few of the subjects talked about.

> I think any developer, no matter the experience, found something useful among the talks. I went with 2 friends which are just starting their JavaScript journey and both left with some extra knowledge and excited about becoming developers.

Reviewer: Nice paragraph! ❤️

My favorite 3 talks were:

1. [Building AI-powered user interfaces](https://www.youtube.com/watch?v=GQjubOMOWzo&list=PLB9NqTp0uKrR8g9ImDK2HPcpuC7oW8wYe&index=11) by Jani Eväkallio

2. [Leveraging (algebraic data) types to make your UI rock solid](https://www.youtube.com/watch?v=HrLPvAOjJxM&list=PLB9NqTp0uKrR8g9ImDK2HPcpuC7oW8wYe&index=15) by Matthias Le Brun

3. [We're All Doing the Best We Can](https://www.youtube.com/watch?v=MtgrsmaKzDg&list=PLB9NqTp0uKrR8g9ImDK2HPcpuC7oW8wYe&index=7) by Jeffrey Lembeck

> I'm not gonna get into each specific once, but I encourage you to check them out on [YouTube](https://www.youtube.com/watch?v=N6qr6L37sl8&list=PLB9NqTp0uKrR8g9ImDK2HPcpuC7oW8wYe).

Reviewer: ```suggestion
I'm not gonna get into each specific one, but I encourage you to check them out on [YouTube](https://www.youtube.com/watch?v=N6qr6L37sl8&list=PLB9NqTp0uKrR8g9ImDK2HPcpuC7oW8wYe).
```

Author: done

> There was also a secondary stage where together with the speakers we would have some casual discussions on the topics. There was a really nice debate about AI and the future of programming now that we have tools like Copilot and ChatGPT. I wish that was recorded...

Reviewer: Can I suggest you rephrase this as: 

```suggestion
During the breaks, there was an alternative stage where together with the speakers we would have some casual discussions on the topics. There was a really nice debate about AI and the future of programming now that we have tools like Copilot and ChatGPT. I wish that was recorded...
```

So anyone reading this doesn't think there are two actual conference stages during the event

Author: Thanks. Indeed, reading it again people might think there are 2 stages.
